src/main/java/org/repodriller/filter/diff/OnlyDiffsWithFileTypes.java

```diff
@@ -0,0 +1,23 @@
+package org.repodriller.filter.diff;
+
+import java.util.List;
+
+/**
+ * Only process diffs on files with certain file extensions.
+ * 
+ * @author Ayaan Kazerouni
+ */
+public class OnlyDiffsWithFileTypes implements DiffFilter {
+
+	private List<String> fileExtensions;
+	
+	public OnlyDiffsWithFileTypes(List<String> fileExtensions) {
+		this.fileExtensions = fileExtensions;
+	}
+	
+	@Override
+	public boolean accept(String diffEntryPath) {
+		return this.fileExtensions.stream().anyMatch(fe -> fe.endsWith(diffEntryPath));
```

Comment:
1. Is this logic backwards? Should it be `diffEntryPath.endsWith(fe)`?

2. If the user provides "m" (MATLAB files), this logic will also capture "asm" (assembly files). You should give an example in the documentation that you require ".m". But I think it's cleaner if you check for `"." + fe` (and then you should document this instead).

Reply:
1. Wow, that's awkward. Thanks for catching that.
2. Sounds good.